components/blog/HeaderLogo.tsx

```diff
@@ -0,0 +1,15 @@
+'use client';
+
+import React from 'react';
+
+import { SymbolIcon } from '@/public/icons';
+
+const HeaderLogo = () => {
+  return (
+    <>
+      <SymbolIcon />
+    </>
+  );
+};
```

Comment:
요렇게 컴포넌트 하나만 리턴하는 경우는!
```suggestion
const HeaderLogo = () => <SymbolIcon />;
```

요래 쓰면 좀 더 간결하게 쓸 수 있어요!!!